### ISSUE TITLE
Feature: (sort-classes) Option to treat function properties as methods

### ISSUE THREAD

**Issue by rmuratov** [azat-io/eslint-plugin-perfectionist#56]
### What rule do you want to change?

sort-classes

### Describe the problem

```typescript
export class Model {
    a: string = ''
    b = () => {
        this.a = 'a'
    }
    c: string = ''
    d() {
        return this.a
    }
}
```

### Code example

```typescript
export class Model {
    a: string = ''
    c: string = ''
    d() {
        return this.a
    }
    b = () => {
        this.a = 'a'
    }
}
```

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Do you think this should be the default behavior or a special option in the rule settings?

**Comment by rmuratov** [comment]
Eventually, I've turned off this rule. I decided that I'd instead sort class fields by hand and semantically, i.e., not all `setThingA`, and `setThingB` in one place, but `setThingA`, `deleteThingA`, `getThingA` in one place.

It feels like sorting class members by simple rules (alphabetical, etc.) works only for simple classes. With that in mind, I now think it's better to leave this rule as is. If someone wants more, they can turn it off and sort manually or use another niche sorting plugin.
